### PR TITLE
Remove `OpCode.CALL_I`

### PR DESCRIPTION
**Pull request by erikzhang**
Closes #144

### CODE REVIEW

File: src/neo-vm/ExecutionContext.cs [hunk 1]
```diff
@@ -71,12 +71,24 @@ public byte[] ScriptHash
...
+            : this(script, callingScriptHash, rvcount, new RandomAccessStack<StackItem>())
+        {
+        }
+
+        private ExecutionContext(Script script, byte[] callingScriptHash, int rvcount, RandomAccessStack<StackItem> stack)
         {
             this.RVCount = rvcount;
             this.Script = script;
+            this.EvaluationStack = stack;
+            this.AltStack = new RandomAccessStack<StackItem>();
```
Comment (shargon) [inline src/neo-vm/ExecutionContext.cs]: I thinkt that we should copy the AltStack too

File: src/neo-vm/ExecutionEngine.cs [hunk 2]
```diff
@@ -1213,17 +1196,22 @@ private bool ExecuteInstruction()
...
+
         public ExecutionContext LoadScript(byte[] script, byte[] callingScriptHash = null, int rvcount = -1)
         {
             return LoadScript(new Script(Crypto, script), callingScriptHash, rvcount);
         }
 
-        protected virtual ExecutionContext LoadScript(Script script, byte[] callingScriptHash = null, int rvcount = -1)
+        private ExecutionContext LoadScript(Script script, byte[] callingScriptHash = null, int rvcount = -1)
         {
             ExecutionContext context = new ExecutionContext(script, callingScriptHash, rvcount);
```
Comment (shargon) [inline src/neo-vm/ExecutionEngine.cs]: We can join this and remove this variable

Comment (erikzhang) [reply src/neo-vm/ExecutionEngine.cs]: What do you mean?

File: src/neo-vm/ExecutionContext.cs (hunk 1 above)
Comment (shargon) [reply src/neo-vm/ExecutionContext.cs]: But is weird to me, the alt stack is isolated and the evaluationStack don't

Comment (shargon) [reply src/neo-vm/ExecutionContext.cs]: Before the stack isolation we reuse both stacks, why now is different?

**Pull request merged**